<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Words</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>5</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(185, 235, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>5</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Level :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>5</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(185, 235, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>5</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Category :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>5</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(185, 235, 232);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>285</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>400</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 173, 170);
color: rgb(255, 255, 255);
padding: 5px;</string>
    </property>
    <property name="text">
     <string>Sentences</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>daasasd</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/play);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/play);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/play);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/play);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>---------&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>70</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/play);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 173, 170);
color: rgb(255, 255, 255);
padding: 5px;</string>
    </property>
    <property name="text">
     <string>Words</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>375</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(116, 173, 170);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>410</width>
      <height>340</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 236, 236);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>400</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>400</width>
      <height>95</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>345</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 8pt &quot;Sansation Light&quot;;</string>
    </property>
    <property name="text">
     <string>english learning software</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>345</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 8pt &quot;Sansation Light&quot;;</string>
    </property>
    <property name="text">
     <string>SDU 2013</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>120</width>
      <height>135</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>135</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_29</zorder>
   <zorder>label_30</zorder>
   <zorder>label_31</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>listWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_32</zorder>
   <zorder>label_33</zorder>
   <zorder>label_34</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label_17</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>label_5</zorder>
   <zorder>label_18</zorder>
   <zorder>label_8</zorder>
   <zorder>label_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>label_19</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_7</zorder>
   <zorder>label_20</zorder>
   <zorder>label</zorder>
   <zorder>label_10</zorder>
   <zorder>label_9</zorder>
   <zorder>label_16</zorder>
   <zorder>label_11</zorder>
   <zorder>label_13</zorder>
   <zorder>label_12</zorder>
   <zorder>label_27</zorder>
   <zorder>label_28</zorder>
   <zorder>label_24</zorder>
   <zorder>label_25</zorder>
   <zorder>label_23</zorder>
   <zorder>label_26</zorder>
   <zorder>label_15</zorder>
   <zorder>label_14</zorder>
   <zorder>label_35</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
